templates/applicantForm.phtml

```diff
@@ -18,6 +18,13 @@
 <?php require 'partials/layout/nav.phtml'; ?>
         <h3>Edit Applicant</h3>
         <form>
+            <section class="stagesDropdown">
+                <label for="stageTitle">Choose Applicants' Stage: </label>
```

Comment:
should this be Applicant's as it's about a single applicant

Reply:
good point u meanie - done it